src/components/Button/Button.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types'
+import './Button.css'
+
+class Button extends Component{
```

Comment:
Why don't we convert this into a simple stateless functional component since we aren't keeping state here nor are we using any lifecycle method.